scripts/quests/bastok/Teak_Me_to_the_Stars.lua

```diff
@@ -0,0 +1,70 @@
+-----------------------------------
+-- Teak Me to the Stars
+-----------------------------------
+-- Log ID: 1, Quest ID: 79
+-- Raibaht : !gotoid 17748012
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.TEAK_ME_TO_THE_STARS)
+
+quest.reward =
+{
+    fame     = 30,
+    fameArea = xi.quest.fame_area.BASTOK,
+    gil      = 2100,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getFameLevel(xi.quest.fame_area.BASTOK) >= 3
+        end,
+
+        [xi.zone.METALWORKS] =
+        {
+            ['Raibaht'] = quest:progressEvent(864, 0, xi.item.GARHADA_TEAK_LUMBER, 2964, 721420799, 40186682, 2, 4095, 262246),
+
+            onEventFinish =
+            {
+                [864] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.METALWORKS] =
+        {
+            ['Raibaht'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:event(871, 0, xi.item.GARHADA_TEAK_LUMBER)
+                end,
```

Comment:
This dialoge should cycle.
Event 501 and event 871 alternate every time you speak with the NPC.

Leave a TODO comment stating this info here, at least. It can be revisited later.

Reply:
Was throwing that out there - is there an example somewhere of equal priority?

Reply:
Check :setPriority() modifier for those functions.  1000 is progress, 100 is event, and I can't remember default off the top of my head (recovering my machine at the moment).  As long as events are equal in priority, they will cycle on their own

Reply:
Thanks!

Reply:
For what its worth.
    Ignore = 1,
    Default = 10,
    ReplaceDefault = 20,
    Message = 50,
    Event = 100,
    Progress = 1000,